gradle.properties

```diff
@@ -3,7 +3,7 @@ kotlin.code.style=official
 # config
 
 # codegen
-smithyVersion=1.23.0
+smithyVersion=1.23.1
```

Comment:
**Suggestion**: Consider using the latest version, **1.24.0**.

Reply:
Wasn't aware of that, updated to latest.